dask/dataframe/tests/test_multi.py

```diff
@@ -299,6 +322,8 @@ def test_sequential_joins():
 
 
 def test_merge_asof_indexed():
+    if DASK_EXPR_ENABLED:
+        pytest.skip("merge_asof not available yet in dask-expr")
```

Comment:
[Based on pytest docs](https://docs.pytest.org/en/6.2.x/skipping.html), I think `xfail` is more appropriate in these situations.

```suggestion
@pytest.mark.xfail(DASK_EXPR_ENABLED, reason="merge_asof not available ye tin dask-expr")
def test_merge_asof_indexed():
```

Reply:
That doesn't work unfortunately, we can't evaluate the config in these skip contexts, don't know why

Reply:
It's the decorator, inline all of them work